docs-examples/example-groovy/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id 'io.micronaut.build.internal.session-examples'
+    id 'groovy'
+}
+
+dependencies {
+    testImplementation(projects.session)
+
+    testImplementation(mn.micronaut.inject.groovy)
+    testImplementation mnTest.micronaut.test.spock
+    testImplementation mn.reactor
+    testImplementation(mn.micronaut.http.client)
+    testImplementation(mn.micronaut.http.server)
+    testImplementation(mn.micronaut.http.server.netty)
+    testImplementation(mnSerde.micronaut.serde.jackson)
+
+    testRuntimeOnly mn.logback.classic
```

Comment:
We can probably extract the commonality for the 3 example dependencies into the session-examples convention plugin?